src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -61,6 +62,11 @@ const SidebarSidecarLayoutContent = ({
 	const currentlyViewedVersion = getVersionFromPath(currentPath)
 	const sidebarProps = sidebarNavDataLevels[currentLevel]
 	const sidebarIsVisible = !isMobileMenuRendered || mobileMenuIsOpen
+	const contentRef = useRef(null)
+
+	const { scrollYProgress } = useScroll({
+		target: contentRef,
```

Comment:
```suggestion
		target: contentRef,
		offset: ["-68px start", "end end"]
```

Sticky elements are not registered during scroll, so the animation doesn't kick in until you've scrolled the distance of the header element which I think is 68px. Is it possible to get that header height value in some fashion?

offset is available as an option in framer-motion useScroll.

Reply:
Ah good call! We can definitely hard-code the `68px` value as a start 💯 We have that value available as a CSS variable, I'll try that out.

The `useScroll` in this PR takes the exact same options as `framer-motion`, since `framer-motion` passes everything through to `@motionone/dom` (at least as far as I can tell!).

Reply:
Sweet okay we had a function for getting the sticky bar height, which is kinda neat - elevated that function to `lib` and am now using it 💯 thanks for mentioning that! 💖